.gitignore

```diff
@@ -401,4 +401,7 @@ spark/utility/tinyxml/Makefile
 spark/utility/tinyxml/Makefile.in
 rcssserver3d/build/*
 spark/build/*
-.svn/*
\ No newline at end of file
+.svn/*
+
+spark/installed/*
+rcssserver3d/installed/*
```

Comment:
should we change our install location so we don't need to do this?

Reply:
I don't see a very strong reason to bother. It would mean everyone would need to delete this folder manually (when this commit makes it into master)

Reply:
yeah ok.  i mean, we could add the delete to setup-simulation.sh, but this is fine, too.